db/data/statements/statements.csv

```diff
@@ -0,0 +1,113 @@
+month,year,deadline_date,payment_date,output_fee,cohort
+
+1,2022,2021-12-25,2022-1-25,true,2021
```

Comment:
I'm not really familiar with these seeds on ECF, but why do we have CSVs here instead of just using the factories? I don't really get the grouping going on here either, I expect there's a pattern but its not very clear just from looking at it 🤔 I wonder if it would be easier to follow if it was just setup in code, that way we could at least add comments?

Reply:
Yeah I'd be eager to avoid CSVs if possible too. Preferably dynamic so they won't need updating every year or two too.